### ISSUE TITLE
latest nvda snapshot

### ISSUE THREAD

**Issue by mjonsson1986** [pumper42nickel/eloquence_threshold#42]
is not compatible with this addon

from the log:
Error while importing SynthDriver eloquence
Traceback (most recent call last):
  File "synthDriverHandler.pyc", line 451, in getSynthList
  File "synthDriverHandler.pyc", line 438, in _getSynthDriver
  File "importlib\__init__.pyc", line 88, in import_module
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1026, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "C:\Users\mattias\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\eloquence.py", line 42, in <module>
    from . import _eloquence
  File "C:\Users\mattias\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\_eloquence.py", line 5, in <module>
    from versionInfo import version_year
ImportError: cannot import name 'version_year' from 'versionInfo' (C:\Program Files (x86)\NVDA\library.zip\versionInfo.pyc)
i'm not a programer
so i have no idea what this error mean
